shorebird_code_push/README.md

```diff
@@ -2,12 +2,30 @@
 
 [![License: MIT][license_badge]][license_link]
```

Comment:
I'd add a discord badge here while you're at it. :)

Reply:
Done.